app/views/catalog/index.json.jbuilder

```diff
@@ -67,6 +67,10 @@ json.included do
     end
   end
 
+  search_fields = blacklight_config.search_fields.values.
```

Comment:
I think it's possible downstream applications have overridden the `search_fields` helper, so the best we can do is wrap it in `Deprecation.silence(Blacklight::ConfigurationHelperBehavior)`?

Reply:
Good point! Wrap this bit of code then? I will try that.